src/topBar/TopBar.js

```diff
@@ -12,16 +15,23 @@ const BoldTypography = styled(Typography)`
 export default function SimpleAppBar(props) {
 
   return (
-    <TopAppBar>
-      <TopAppBarRow>
-        <TopAppBarSection>
-          <TopAppBarTitle>
-            <BoldTypography use="headline4">
-              Olimpijczyk
-            </BoldTypography>
+      <TopAppBar>
+        <TopAppBarRow>
+          <TopAppBarSection>
+            <TopAppBarTitle>
+              <BoldTypography use="headline4">
+                Olimpijczyk
+              </BoldTypography>
             </TopAppBarTitle>
-        </TopAppBarSection>
-      </TopAppBarRow>
-    </TopAppBar>
-  );
-}
\ No newline at end of file
+          </TopAppBarSection>
+          <TopAppBarSection alignEnd>
+            <IconButton
+              icon={github}
+              tag="a"
+              target="_blank"
+              href={'https://github.com/pzmudzinski/react-swimming-pool-tracker'}/>
```

Comment:
It can be simply `href="https://github.com/pzmudzinski/react-swimming-pool-tracker"`